src/stacks-hierarchy/Root_TicketAssistantStack/Root_TicketAssistantStack.tsx

```diff
@@ -29,14 +30,23 @@ type Props = RootStackScreenProps<'Root_TicketAssistantStack'>;
 export const Root_TicketAssistantStack = ({navigation}: Props) => {
   const [activeTab, setActiveTab] = useState(0);
   const {bottom: safeAreaBottom} = useSafeAreaInsets();
+  const analytics = useAnalytics();
   const {theme} = useTheme();
   const [previousTab, setPreviousTab] = useState<any>();
   return (
     <TicketAssistantContextProvider>
       <FullScreenHeader
         leftButton={
           activeTab === 0
-            ? {type: 'close'}
+            ? {
+                type: 'close',
+                onPress: () => {
+                  analytics.logEvent(
+                    'Ticketing',
+                    'Ticket assistant closed from welcome screen',
+                  );
+                },
```

Comment:
I would think this replaced the standard close action in the header, and would now only log the analytics event?

Reply:
Yes, saw at the same time as you. Should be fixed now